getting-started/installation/general-installation/installation-community.markdown

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Installing Community
 published: true
-sorting: 50
```

Comment:
@me-coder was this a suggestion from someone on our team? @olehermanse @nickanderson other than this change that doesn't seem needed I think we are good to commit!